docs/Contribution.md

```diff
@@ -0,0 +1,104 @@
+# Contribution Guide
+
+Thank you for your interest in contributing to Mobile App Control Center, Systers! There are always bugs to file; bugs to fix in the code; improvements to be made to the documentation; and more.
+
+
+The instructions given below are for software developers who want to work on [our current code](https://github.com/systers/macc).
+
+## Git workflow
+
+When you want to start contributing, you should [follow the instructions](https://github.com/systers/macc/blob/develop/docs/Installation%20Guide.md). After that, follow these steps.
+
+Set your cloned fork to track upstream changes (changes to the main repository), then fetch and merge from the upstream branch:
+
+    `$ git remote add --track upstream https://github.com/systers/macc.git`
+    `$ git fetch upstream develop`
+    `$ git merge upstream/develop`
+
+
+Set up a branch for a particular set of changes and switch to it:
+
+    `$ git checkout -b my_branch`
+    `$ git checkout my_branch`
+
```

Comment:
And maybe you should also mention that a request can be pulled without creating a new branch explicitly. That would be 1 request at a time, but its still a good option I think. :)